Autopilot/Src/MPXV7002DP.cpp

```diff
@@ -41,19 +39,13 @@ uint16_t MPXV7002DP::GetAverage() {
 }
 
 void MPXV7002DP::GetResult(airspeedData_t &Data) {
-    if (! dataNew)
-    {
-        Data.isDataNew = false;
-        return;
-    }
-    Data.airspeed = airspeed;
-    Data.isDataNew = dataNew;
+    Data.airspeed = GetAirspeed();
+    Data.isDataNew = true;
     Data.sensorStatus = sensorStatus;
-
-    dataNew = false;
 }
 
 void MPXV7002DP::Begin_Measuring() {
+    
     airspeed = GetAirspeed();
     dataNew = true;
 }
```

Comment:
We don't need this anymore, right? If so, remove it and refer to my comment in `airspeed.hpp`